### PR TITLE
ZEPPELIN-2562 Make Add paragraph button more prominent

### PR DESCRIPTION
**Pull request by malayhm**
### What is this PR for?
- Earlier add paragraph button was tiny plus between two paragraphs. 
- With this change, we have added `Add Paragraph` text along with `+` and increased the height without any flicker so that user will get to know about adding a paragraph easily

### What type of PR is it?
[Improvement]

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2562

### How should this be tested?
1. Build: mvn clean package -Denforcer.skip -DskipTests -Drat.skip
2. Mouse hover over the empty space between the paragraph to create a paragraph

### Screenshots (if appropriate)

Before:
add-para-before

After:
add-paragraph-after

### Questions:
* Does the licenses files need update? - No
* Is there breaking changes for older versions? - No
* Does this needs documentation? - No

### CODE REVIEW

**Comment by 1ambda** [comment]
Cool!

**Comment by Leemoonsoo** [comment]
Tested and it works well!
@malayhm Could you try restart a failed CI task? https://travis-ci.org/malayhm/zeppelin/jobs/272238226

File: zeppelin-web/src/app/notebook/notebook.html [hunk 1]
```diff
@@ -133,8 +136,8 @@ <h4 class="plus-sign">&#43;</h4>
          ng-hide="currentParagraph.config.tableHide && viewOnly"
          ng-dblclick="paragraphOnDoubleClick(currentParagraph.id);">
     </div>
-    <div class="new-paragraph" ng-click="insertNew('below');" ng-hide="!$last || viewOnly || asIframe || revisionView">
-      <h4 class="plus-sign">&#43;</h4>
+    <div class="new-paragraph last-paragraph" ng-click="insertNew('below');" ng-hide="!$last || viewOnly || asIframe || revisionView">
+      <h4 class="plus-sign">&#43; Add Paragrap</h4>
```
Comment (1ambda) [inline zeppelin-web/src/app/notebook/notebook.html]: Typo: Add Paragrap`h`

Comment (malayhm) [reply zeppelin-web/src/app/notebook/notebook.html]: Fixed the typo.

**Comment by 1ambda** [comment]
Looks good to me

**Comment by malayhm** [comment]
@1ambda Can we merge this to master if there are no more discussions?

**Comment by malayhm** [comment]
@1ambda Sure.

**Comment by 1ambda** [comment]
Merge if no more discussion.

**Pull request closed without merging**